packages/web-components/tests/e2e-storybook/cypress/integration/masthead/masthead-default.e2e.js

```diff
@@ -60,12 +68,12 @@ describe('dds-masthead | default (desktop)', () => {
   });
 
   it('should support custom url for IBM logo', () => {
-    cy.intercept(`https://1.www.s81c.com/common/carbon-for-ibm-dotcom/translations/masthead-footer/v2/*`, {
+    cy.intercept(`https://1.www.s81c.com/common/carbon-for-ibm-dotcom/translations/masthead-footer/+(v2|v2.1)/*`, {
```

Comment:
Is this some kind of fancy cypress regexing? Can you link to documentation for this?

Reply:
Wham: https://docs.cypress.io/api/commands/intercept#Glob-Pattern-Matching-URLs